Gruntfile.js

```diff
@@ -34,7 +34,7 @@ module.exports = function(grunt) {
   grunt.registerTask('test', [
     'build-tests',
     'mochaTest',
-    'lint',
+    'eslint',
```

Comment:
I think this patch is still lingering because it was removed at the end of your work done on https://github.com/mozilla/web-ext/pull/434 You can manually fix it just by editing this line back to the way it is on master.